_topic_map.yml

```diff
@@ -531,10 +531,27 @@ Topics:
     Distros: openshift-*
   - Name: Garbage Collection
     File: garbage_collection
-    Distros: openshift-*
-  - Name: Scheduler
-    File: scheduler
+    Distros: openshift-*    
+  - Name: Advacnced Scheduling
```

Comment:
@aheslin I have moved these topics around so many times (maybe the cause of the following bad links!) that a second opinion on the organization would be most appreciated.